CHANGELOG.md

```diff
@@ -2,6 +2,25 @@ faucet-pipeline-core version history
 ====================================
 
 
+v1.5.0
+------
+
+_2020-04-20_
+
+notable changes for end users:
+
+* configuration can now be asynchronous:
+
+  ```javascript
+  module.exports = new Promise(…);
+  ```
```

Comment:
Is this sufficiently grokkable?

Reply:
I think, I would write:

```js
module.exports = async function() {
  ...
}
```

Reply:
Good point, but I decided to sidestep the issue:

> configuration can now be asynchronous, i.e. `faucet.config.js` may export a promise, allowing for dynamically generated configuration values